CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Overview
+
+This documents explains the processes and practices recommended for
+contributing enhancements to the Tempo bundle.
+
+- Generally, before developing enhancements to this charm, you should consider
+  [opening an issue](https://github.com/canonical/tempo-bundle) explaining
+  your use case.
+- If you would like to chat with us about your use cases or proposed
+  implementation, you can reach us on the
+  [Charmhub Mattermost](https://chat.charmhub.io/charmhub/channels/charm-dev)
+  or [Discourse](https://discourse.charmhub.io/).
+- All enhancements require review before being merged.
+  Apart from code quality and test coverage, the review will also take into
+  account the Juju administrator user experience using the bundle.
+
+## Development
+
+There are two charms used for development:
+- tempo
+- s3proxy
+
+### Deploy with local charms
+
+To deploy the bundle using only/some local charms you need to render the
+[`bundle.yaml.j2`](bundle.yaml.j2) template and then deploy the rendered bundle
+as usual.
+
+#### Render template using the rendering script
+You can render and deploy a production bundle using:
+
+```shell
+# generate and activate a virtual environment with dependencies
+tox -e integration --notest
+source .tox/integration/bin/activate
+
+./render_bundle.py bundle.yaml
+juju deploy ./bundle.yaml
+```
+
+Now `juju switch` into the newly created model (or use `--model=MODEL` in
```

Comment:
tox -e integration --notest will not create any model.

 ```suggestion
Now `juju add-model foo` and deploy the bundle with `juju deploy ./bundle.yaml`
```

do we need `--trust` too perhaps?

Reply:
also add a link to a minio deployment guide as @mmkay suggested